packages/svelte/src/internal/client/dom/blocks/each.js

```diff
@@ -258,6 +266,8 @@ function reconcile_indexed_array(array, state, anchor, render_fn, flags) {
 		value = array[i];
 		item = a_items[i];
 		b_items[i] = item;
+		// Indexed each blocks use the index as the key
+		item.k = i;
```

Comment:
now that i think about it, how could this ever have changed? the index is fixed upon creation. this line seems unnecessary

```suggestion
```